command/ad_info.go

```diff
@@ -0,0 +1,133 @@
+package command
```

Comment:
A project’s `cmd` directory is normally where its applications live. Code that can be imported into other projects should be moved out of `cmd` and into other package directories. It would be unusual to see an import of a package called `cmd `.

So, `command` is bad for the same reason, but not as standard in use. So, let's do one of the following:

1. Rename `command` to `libcli` (or something similar if you have anything better)
2. Or, rename `command` to `internal` so that there is no importing the command logic into other projects.